src/fhir/1_Condition_DCW_JT_0.json

```
@@ -0,0 +1,38 @@
+{
+  "resourceType": "Condition",
+  "clinicalStatus": {
```

To build an idempotent transaction Bundle, the upload.py script needs each resource to have a consistent ID. ([at least as implemented](https://github.com/uwcirg/asbi-screening-app/blob/master/src/fhir/upload.py#L46-L47))

```suggestion
  "id": "condition-hypertension-1",
  "clinicalStatus": {
```